docs/proposals/remote-artifacts.md

```diff
@@ -0,0 +1,267 @@
+<!--
+Copyright The Shipwright Contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+---
+title: remote-artifacts
+authors:
+  - "@otaviof"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-10-02
+last-updated:  2020-10-02
+status: provisional
+---
+
+Shipwright Remote Artifacts
```

Comment:
I have a conceptual issue with the word artifacts. We already have "artifacts" that are remote, like "github" repositories.